benchexec/tools/cpa-witness-exec.py

```diff
@@ -0,0 +1,180 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+# prepare for Python 3
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import subprocess
+import sys
+import os
+
+import benchexec.result as result
+import benchexec.util as util
+import benchexec.tools.template
+from benchexec.model import SOFTTIMELIMIT
+
+
+class Tool(benchexec.tools.template.BaseTool):
```

Comment:
There is much in common with the CPAchecker-tool-wrapper.
Would it be possible to merge those two?
Or inherit this from the other tool?